mesa/model.py

```diff
@@ -21,7 +22,16 @@ class Model:
     def __new__(cls, *args: Any, **kwargs: Any) -> Any:
         """Create a new model object and instantiate its RNG automatically."""
         obj = object.__new__(cls)
-        obj._seed = kwargs.get("seed", None)
+        obj._seed = kwargs.get("seed")
+        if obj._seed is None:
+            # See https://docs.python.org/3/library/random.html#random.seed
```

Comment:
The comment is not 100% accurate, but I think it can be removed completely anyway. I don't think the rational is important here (it should go into the commit message if you want to preserve it).

Reply:
I can make it more concise. But I think a new rule of thumb is that if ChatGPT can't deduce the purpose of the code that matches this comment, then the comment is needed.

Reply:
Also, having to view the blame for a small helpful comment is inconvenient.